contribute/developer-guide.md

```diff
@@ -224,57 +224,19 @@ make test-go-integration-postgres
 
 ### Run end-to-end tests
 
-Grafana uses [Cypress](https://www.cypress.io/) to end-to-end test core features. Core plugins use [Playwright](https://playwright.dev/) to run automated end-to-end tests. You can find more information on how to add end-to-end tests to your core plugin [in our end-to-end testing style guide](./style-guides/e2e-plugins.md)
+- Grafana uses [Playwright](https://playwright.dev/) to run automated end-to-end tests. You can find more information [in our end-to-end testing style guide](./style-guides/e2e-playwright.md#playwright-for-plugins)
 
-#### Run Cypress tests
+- Each version of Playwright needs specific versions of browser binaries to operate. You need to use the Playwright CLI to install these browsers: `yarn playwright install chromium`.
+- Run tests with `yarn e2e:playwright [optional path to test file]`.
 
-To run all tests in a headless Chromium browser.
+- To open the las HTML, you can run `yarn playwright show-report`. You can also open an arbitrary report with `yarn playwright show-report <reportLocation>`. For Grafanistas, the reports are also downloable from CI by:
```

Comment:
>  For Grafanistas, the reports are also downloable from CI by:

I believe thse are accessable for all.

Reply:
ah tbf i couldn't remember if everyone had access or not 🤷

Reply:
But is it just for Grafanistas?

Reply:
Ah, ok, so I'll remove that bit. Thanks